### DISCUSSION TITLE
Pricing is ridiculous

### DISCUSSION THREAD

**Discussion by osseonews** [oramasearch/orama#713] [General]
Why did you change your pricing so radically? It's either free or then jumps to $499 a month? Really? This is a bad business decision that will greatly limit your appeal. You need more in the middle plans, like $25 per month, like you had previously. Nobody wants to get locked into a platform where the first paid tier is $500 a month.

**Comment by micheleriva** [comment]
Hi @osseonews, you may have noticed that we dumped the former PRO plan into the free one. You're getting a former $300 plan for free now. I think that's good honestly!

**Reply by osseonews** [reply]
Maybe, but I really have no idea what is included now and before, and now I'm concerned you will do another switcheroo out of nowhere and the free plan will be ditched in some fashion. Plus, not sure how a startup Saas business whose 1st paid tier is $500 a month is going to make it. It's not a viable pricing structure. 

When I am considering a service to go to production, I like to see how much it might cost and I'd like to know the company will be around in a few years. I certainly don't want to lock into a platform for years knowing that if I had to pay, I'm jumping up to $500 a month.  And I don't want to build something thinking that the company doesn't have a viable financial plan.

The right pricing model is what you had before, a free tier, and then a mid-tier ($30 a month seems fair given competitor pricing), and then you can jump up from there. Maybe offer a bit more on a new $30 a month plan and a bit less on a free plan.

**Comment by osseonews** [comment]
To follow up on my last post, the way you present your pricing, gives signals to a potential customer. I look at your pricing page and what it signals to me is that this company wants to be enterprise only. Maybe I'm wrong, but that's the vibe. And if I see that, as a small business owner and start up, there is no way I want to use the product because I have no interest in an enterprise-only product and if I did, I would just use. Why not look at how other search companies present pricing, like Elastic or Algolia. These companies have become quite successful. I have used Algolia for many years.  

And actually I think your product is amazing. I tested many other paths to Hybrid Search and AI generated docs (we run on Docusaurus) and your system actually works.  But, then I go back to the pricing page, and it's like what the heck happened here.
